docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -197,7 +197,26 @@ implementation is used:
   - Set to `false` to disable in-memory caching of metadata files on the 
     coordinator. This cache is not used when `fs.cache.enabled` is set to true.
   - `true`
-
+* - `iceberg.expire_snapshots.min-retention`
+  -  Minimal retention period for `expire_snapshot` procedure. Equivalent
```

Comment:
```suggestion
  -  Minimal retention period for the `expire_snapshot` procedure. Equivalent
```

Reply:
Also .. can we link to the procedure docs and have an example there? Also ... we should link to the duration property type